src/config.py

```diff
@@ -69,6 +73,18 @@ def spark_conf(self):
         """Return the dict representation of the configuration file."""
         s3_log_dir = self.s3_log_dir
         conn_config = self.s3_creds_client.get_s3_connection_info()
+
+        ingress_pattern = re.compile(r"http://.*?/")
```

Comment:
As we discussed, we need to understand how to properly handle validation of this field.